wallet/src/de/schildbach/wallet/service/CoinJoinService.kt

```diff
@@ -388,7 +388,7 @@ class CoinJoinMixingService @Inject constructor(
         if (mode != CoinJoinMode.NONE && this.mode == CoinJoinMode.NONE) {
             configureMixing()
         }
-        updateBalance(walletDataProvider.wallet!!.getBalance(Wallet.BalanceType.AVAILABLE))
+        updateBalance(walletDataProvider.getWalletBalance())
```

Comment:
use the last calculated balance instead of recalculating it.  Also use the total balance, rather than AVAILABLE, which will not include unconfirmed TX's.